lib/companies/companies.ex

```diff
@@ -134,8 +137,9 @@ defmodule Companies.Companies do
 
   defp predicates(query, %{"type" => "hiring"}) do
     query
-    |> join(:inner, [c], j in assoc(c, :jobs))
-    |> order_by([_c, j], desc: j.inserted_at)
+    |> exclude(:left_join)
```

Comment:
Since we're going to apply an inner join now, let's remove the left join applied in the base query build up.